src/blocks/custom/form/components/form-buckaroo-options.js

```diff
@@ -0,0 +1,81 @@
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import { SelectControl, TextControl, TextareaControl } from '@wordpress/components';
+
+
+export const FormBuckarooOptions = (props) => {
+  const {
+    service,
+    emandateDescription,
+    redirectUrl,
+    redirectUrlCancel,
+    redirectUrlError,
+    redirectUrlReject,
+    onChangeService,
+    onChangeEmandateDescription,
+    onChangeRedirectUrl,
+    onChangeRedirectUrlCancel,
+    onChangeRedirectUrlError,
+    onChangeRedirectUrlReject,
+  } = props;
+
+  const buckarooOptions = [
+    { label: 'iDEAL', value: 'ideal' },
+    { label: 'Emandate', value: 'emandate' },
+  ];
+
+  return (
+    <Fragment>
+      {onChangeService &&
+        <SelectControl
+          label={__('Service', 'eightshift-forms')}
+          help={__('Please select which Buckaroo service you wish to use', 'eightshift-forms')}
+          value={service}
+          options={buckarooOptions}
+          onChange={onChangeService}
+        />
+      }
+
+      {onChangeEmandateDescription && service === 'emandate' &&
+        <TextareaControl
+          label={__('Emandate description', 'eightshift-forms')}
+          value={emandateDescription}
+          help={__('A description of the (purpose) of the emandate. This will be shown in the emandate information of the customers\' bank account. Max 70 characters.', 'eightshift-forms')}
+          onChange={(newValue) => {
+            const MAX_CHARS_IN_FIELD = 70;
```

Comment:
This constant must be here? Can't it be defined outside the `return` statement, and used here?